packages/cipherstash-proxy/src/postgresql/frontend.rs

```diff
@@ -86,7 +86,7 @@ where
         let sent: u64 = bytes.len() as u64;
         counter!(CLIENTS_BYTES_RECEIVED_TOTAL).increment(sent);
 
-        if self.encrypt.is_passthrough() {
+        if self.encrypt.config.mapping_disabled() {
```

Comment:
`is_passthrough` checks if mapping is disabled or encrypt config is empty. Here it's updated to only one